docs/stdlib/numbers.rst

```diff
@@ -407,11 +425,11 @@ from :eql:type:`str` and :eql:type:`json`.
 
     :index: floor divide division
 
-    Floor division.
+    Performs a floor based divison between two arbitrary numbers.
 
-    The result is rounded down to the nearest integer. It is
-    equivalent to using regular division and the applying
-    :eql:func:`math::floor` to the result.
+    The result of this operation is rounded down to its nearest integer.
+    It is the equivalent to using regular division and then applying
+    :eql:func:`math::floor` to the result:
```

Comment:
```suggestion
    In floor-based division, the result of a standard division operation is
    rounded down to its nearest integer. It is the equivalent to using regular
    division and then applying :eql:func:`math::floor` to the result.
```

- The result of this operation has already been rounded, so I attempted to clarify the first sentence to read that the result of this is the rounded down result of standard division.
- Removed the colon since the following code block is not introduced by this sentence in particular. We could add "Examples:" if you want, but I think it will be understood without that.